### ISSUE TITLE
[http client] communication error: 400 Bad Request, Protocol not supported

### ISSUE THREAD

**Issue by SteadEXE** [glpi-project/glpi-agent#66]
### Documentation

Yes, I read it

### Solution search

Yes, I read it

### Is your help request related to a problem?

Sorry to bother you but my agent keeps complaining about :
[http client] communication error: 400 Bad Request, Protocol not supported

When trying to run a deploy task

I don't know what is wrong

### Expected behavior

It should ask the server for task, and do nothing, since there are no task deployed, like it does with fusion inventory

### Actions you've considered

I tried to find what is wrong with my installation or current configuration, without success...

### Additional context

I am using GLPI 10.0.0-rc1 with GLPI Inventory 1.0.0-rc4
Server is running with FastCGI behind IIS.

**Comment by g-bougard** [comment]
Hi @SteadEXE 
you forgot to tell which agent version you're using. If not, please upgrade to v1.1 just released last friday.
Assuming you server is located at `http://glpi-server/`, on agent side, be sure to define `server` parameter to `http://glpi-server/marketplace/glpiinventory` if you installed plugin via Marketplace, or use `http://glpi-server/plugins/glpiinventory` if you installed plugin manually into `/plugins` glpi subfolder.

**Comment by SteadEXE** [comment]
I am using GLPI Agent 1.1 (from friday). I think you gave me the clue, I made it point to front/inventory.php.

**Comment by SteadEXE** [comment]
With the format : http://glpi-server/plugins/glpiinventory
I get : 

[Tue Feb  8 15:36:46 2022][info] sending prolog request to server0
[Tue Feb  8 15:36:46 2022][error] [http client] communication error: 405 Method Not Allowed

**Comment by SteadEXE** [comment]
Okay it's propably because on IIS, when you post/put or whatever on the root of a directory, IIS uses a different handler.
My work around was to use http://glpi-server/plugins/glpiinventory/index.php

It could be nice to mention it inside the documentation maybe.

Now it works 👍

**Comment by SteadEXE** [comment]
For people having the same issue, I finally managed to get rid of the error without having to add "index.php" to the URL.
When GLPI is behind a IIS, StaticFile handler is used before PHPFastCgi when there is no .php file in the path.

To resolve the issue I edited the Handler Mappings -> StaticFile handler -> Restriction -> Verbs.
Instead of "All" I used the second options which allows you to list verbs. My list is "GET,POST,PUT,PATCH,DELETE,OPTIONS,HEAD".

![image](https://user-images.githubusercontent.com/1759156/211764160-7908896a-719a-4139-8720-d8048a2c5a65.png)

Hope it can help.
